server/repository/src/db_diesel/vaccine_course/vaccine_course_dose_row.rs

```diff
@@ -26,6 +30,12 @@ table! {
 
 joinable!(vaccine_course_dose -> vaccine_course (vaccine_course_id));
 allow_tables_to_appear_in_same_query!(vaccine_course_dose, vaccine_course);
+allow_tables_to_appear_in_same_query!(vaccine_course_dose, clinician_link);
+allow_tables_to_appear_in_same_query!(vaccine_course_dose, clinician);
+allow_tables_to_appear_in_same_query!(vaccine_course_dose, name_link);
+allow_tables_to_appear_in_same_query!(vaccine_course_dose, name);
+allow_tables_to_appear_in_same_query!(vaccine_course_dose, item_link);
+allow_tables_to_appear_in_same_query!(vaccine_course_dose, item);
```

Comment:
These were in the vaccination row file, figured they should probably live with the relevant table...